packages/cli/lib/install.ts

```diff
@@ -155,8 +155,7 @@ async function runAddon({ addon, multiple, workspace, workspaceOptions }: RunAdd
 				fileContent = content(fileContent);
 				if (!fileContent) return fileContent;
 
-				// TODO: fix https://github.com/rolldown/tsdown/issues/575 to remove the `replaceAll`
-				writeFile(workspace, path, fileContent.replaceAll('<\\/script>', '</script>'));
```

Comment:
This is still needed
We can still remove the test that is skipped as new snapshot tests covers well this.

Reply:
Oh yeah, i had something wrong in mind. Will re-add over the next few days.